modules/Json.tla

```diff
@@ -86,6 +86,13 @@ JsonDeserialize(absoluteFilename) ==
   (*************************************************************************)
   CHOOSE val : TRUE
 
+JsonDeserializeWithNull(absoluteFilename, nullValue) ==
```

Comment:
What is `nullValue` for?

Reply:
I've learned that in TLA+ we don't have the concept of null, right? Tell me if I'm mistaken (or if something new was introduced which allows it).

When you use a deserialization operator, you may want to handle nulls. When it's not handled, it just returns garbage (like `%2%` in my terminal). So I've thought that the user may have a `null` representation which he whishes to use, like a default value (or replacement for nulls). See example below.

```ruby
# Let's say we have a `my-json.json` file with the contents `[{"amount": 10}, {"amount": null}]`
# We will run some assertions below (all are `TRUE`)
<<[amount |-> 10], [amount |-> 0]>>      = JsonDeserializeWithNull("my-json.json", 0)

<<[amount |-> 10], [amount |-> "null"]>> = JsonDeserializeWithNull("my-json.json", "null")

<<[amount |-> 10], [amount |-> <<>>]>>   = JsonDeserializeWithNull("my-json.json", <<>>)
```

There is a test which uses the fixture file (have a better name for this type of file?) `json-with-null.json` where I replace the default value with a set.

Reply:
I will add comments to the tla and java files if these operators are desired, but I can delete too if you if you feel like it =D

Reply:
I now realize that you only propose it for *de*-serialize (indicating that the ergonomics of the `Json*` operator names is not ideal).  Do you have a concrete use-case in mind where the Json input has `null` values with which we could validate the `*WithNull` operator? For example, we could also consider passing in a `LAMBDA` to give users more control.  At any rate, I suggest moving the `*WithNull` enhancement to a separate issue.

Reply:
Got it, I am gonna do that later in the day. Thanks. 

The point is that we return null value from the deserialization, but it does not work correctly. Maybe we could not handle Json null values for now until we have a solution.

Reply:
Removed.